en/python_introduction/prompt.md

```diff
@@ -15,3 +15,12 @@ Python {{ book.py_release }} (...)
 Type "help", "copyright", "credits" or "license" for more information.
 >>>
 ```
+
+<!--sec data-title="Opening: RunCode Cloud Environment" data-id="runcode_prompt" data-collapse=true ces-->
+
+To open the Python terminal on RunCode, go to Workspaces → New Workspace → Jupyter Lab. This will open a Python prompt 
+which is depicted by `>>>`. If you already opened an Ubuntu terminal open, in the [Intro to Command Line](../intro_to_command_line/README.md) chapter, you can just type `python3` in the prompt as shown below to open a Python prompt.
+
+`ubuntu@runcode:~$ python3`
+
+<!--endsec-->
```

Comment:
My final comment would just be that if we find there are lots of places where we need to keep editing sections to say, "if you followed the installation guide do this" then "if you're using RunCode do this", then it might be worth looking at creating two seperate versions of the tutorial for each method. However, if this is minimal, you can leave as it is, as long as we're clear when we reach instructions that these will be diffent depending on setup upfront.

Reply:
Where different instructions are alternatives to each other, we could probably make that much clearer by having them in a tab-headed section rather than in an expandable section. We've discussed this possibility some time in the past (I'll have to look up the issues or PRs about that), but I think that was back when we used GitBook for the tutorial, so we'd have to see whether according features or plugins are available for Honkit, which is now used to render the tutorial.

Reply:
> I'll have to look up the issues or PRs about that

- #188
- https://github.com/DjangoGirls/tutorial/issues/862#issuecomment-247970100
- #1173

Reply:
I have moved all RunCode setup instructions to one file to make a separate installation guide as is done for the Chromebook and the only place where the RunCode instructions had to be left was for opening the command line. I felt it just has to be there as that section is also part of the `Introduction to Command Line` which all users need to go through. I avoided making those comments in many sections so I just added notes on sections that RunCode users need to skip so which should make the instructions clearer.

This seemed easier than reading about tab-headed sections and changing the whole tutorial until we decided to change to that.